app/assets/stylesheets/fortitude/components/_box.scss

```diff
@@ -4,7 +4,7 @@ title: Box
 name: box
 category: Objects
 ---
-The box object simply boxes off content.
+The box object allows you to separate different sections of content, and through extensions customize things like background colors, border colors, and border widths.
```

Comment:
the responsive border utilities generally should be used instead for border-width